compiler/circle2circle/src/Circle2Circle.cpp

```diff
@@ -167,6 +202,22 @@ int entry(int argc, char **argv)
   std::string input_path = arser.get<std::string>("input");
   std::string output_path = arser.get<std::string>("output");
 
+  if (arser["--sparsify_tensor"])
+  {
+    options->enable(Algorithms::SparsifyTensorPass);
+    options->param(AlgorithmParameters::Sparsify_tensor_name,
+                   arser.get<std::string>("--sparsify_tensor"));
+    options->param(AlgorithmParameters::Sparsify_traversal_order,
+                   arser.get<std::string>("--sparsify_traversal_order"));
+    options->param(AlgorithmParameters::Sparsify_format,
+                   arser.get<std::string>("--sparsify_format"));
+    if (arser["--sparsify_block_size"])
+      options->param(AlgorithmParameters::Sparsify_block_size,
+                     arser.get<std::string>("--sparsify_block_size"));
+    options->param(AlgorithmParameters::Sparsify_block_map,
+                   arser.get<std::string>("--sparsify_block_map"));
+  }
```

Comment:
Can this block work when only `sparsify_tensor` option is provided?

Reply:
@seanshpark Yes. But it would be the tensor far from what user think it would be. Actually, it only supports block-sparsification. It should be expanded to general sparsification later.